core/meta/inc/TInterpreter.h

```diff
@@ -251,6 +251,7 @@ class TInterpreter : public TNamed {
    virtual void   CodeComplete(const std::string&, size_t&,
                                std::vector<std::string>&) {;}
    virtual int Evaluate(const char*, TInterpreterValue&) {return 0;}
+   virtual std::string Evaluate(const char*) {return 0;}
```

Comment:
That's terminating, you cannot construct a `std::string` from `0`! Return `{}` or `""` instead.